scripts/modules/config-helpers.js

```
@@ -0,0 +1,588 @@
+
+	if (actualType === expectedType) {
+		return value;
+	}
+
+	// Attempt type conversion for common cases
+	try {
+		switch (expectedType) {
+			case 'string':
+				return String(value);
+			case 'number':
+				const num = Number(value);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)